graphrag/index/operations/extract_entities/strategies/graph_intelligence.py

```diff
@@ -106,7 +106,7 @@ async def run_extract_entities(
             )
 
     entities = [
-        ({"name": item[0], **(item[1] or {})})
+        ({"title": item[0], **(item[1] or {})})
```

Comment:
WIll this change require a re-index?

Reply:
Nope! I just moved it all the way to the original source instead of renaming in the final dataframe, so it's less confusing to track